### DISCUSSION TITLE
deposit did not arrive

### DISCUSSION THREAD

**Discussion by marbush1** [Chia-Network/chia-blockchain#17406] [Support]
Hello, 
that is my transaction ID, from mexc to chia wallet. I can see it in xchscan but nothing in my wallet 0x2ec7f142323a114f229b7e672588ea177ac5abd5be803297c5ef6ba89f0c31b8
what should I do? thanks in advance

**Comment by BrandtH22** [comment]
Hey @marbush1 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

The first thing I would check is that the wallet is connected to peers and synced (indicated by the icon in the top right, if green then it is synced). 

If it is synced then I would try a resync using the GUI settings / advanced / resync option. This will close the GUI and upon reopening will resync the wallet from scratch.

If it is not synced then I would verify you are connected to peers by clicking that wallet icon in the top right. If connected to peers but not syncing fully then the first thing to check is that your system time is correct. I like to use this site to verify https://time.is/

If the system time is correct then I would look for errors or warnings in the logs at `~\.chia\mainnet\log` and report any found here so we can better assist 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
